### ISSUE TITLE
Login fails on CORS error with /session endpoint

### ISSUE THREAD

**Issue by gibsonf1** [inrupt/solid-client-authn-js#996]
After updating the library this morning, we were no longer able to login to TrinPod because of a CORS error on the new /session endpoint of the login sequence.  We fixed the issue by adding a /session enpoint to TrinPod that now returns a CORS response.

```
Access to fetch at 'http://frederick.localhost:8333/session' from origin 'http://localhost:8080' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```

**Comment by NSeydoux** [comment]
Hi @gibsonf1 , and thanks for reporting this. Could you give a try to the fix implemented in https://github.com/inrupt/solid-client-authn-js/pull/999, to check that it prevents the failure you experimented ?

**Comment by Vinnl** [comment]
To clarify, you can try it out using:

```
    npm install @inrupt/solid-client-authn-browser@fixsession-endpoint-cors-issue
```

**Comment by NSeydoux** [comment]
@gibsonf1 We just released v1.6.0 of the library, which replaces the fix that caused this issue initially with a longer term solution. The `/session` endpoint is no longer looked up, and the "logout-on-refresh" issue should be resolved on both ESS and NSS.
